app/views/layouts/exp_atom/_editor.html.haml

```diff
@@ -23,25 +23,24 @@
     %fieldset
       .toolbar-pf-actions
         .form-group
-          %button
-            - t = _('Commit expression element changes')
-            = link_to(image_tag(image_path('toolbars/commit.png'), :alt => t),
-              {:action => 'exp_button', :pressed => 'commit'},
-              "data-miq_sparkle_on"  => true,
-              "data-miq_sparkle_off" => true,
-              :remote                => true,
-              "data-method"          => :post,
-              :title                 => t)
-          %button
-            - t = _("Discard expression element changes")
-            = link_to(image_tag(image_path('toolbars/discard.png'), :alt => t),
-              {:action => 'exp_button', :pressed  => 'discard'},
-              "data-miq_sparkle_on"  => true,
-              "data-miq_sparkle_off" => true,
-              :remote                => true,
-              "data-method"          => :post,
-              :title                 => t)
-
+          - t = _('Commit expression element changes')
```

Comment:
@hstastna Here you go:

%button.btn.btn-default{:title => t,
"data-miq_sparkle_on" => true,
:remote               => true,
"data-method"         => :post,
:onclick => "miqAjaxButton('#{url_for(:action => 'exp_button', :pressed => 'commit')}');"}
%i.fa.fa-check
- t = _("Discard expression element changes")
%button.btn.btn-default{:title => t,
"data-miq_sparkle_on" => true,
:remote               => true,
"data-method"         => :post,
:onclick => "miqAjaxButton('#{url_for(:action => 'exp_button', :pressed => 'discard')}');"}
%i.pficon.pficon-close